### PR TITLE
reproduce #11 by forcing natives version

### PR DESCRIPTION
**Pull request by michaelglass**
using node 10, I always get the warning from #11, however the hard failure only happens when using natives 1.1.2

this PR reproduces #11's failure by forcing usage of natives 1.1.2.

### CODE REVIEW

File: test_app/package.json [hunk 1]
```diff
@@ -18,6 +18,6 @@
     "echoMe": "bin/echoMe"
   },
   "dependencies": {
-    "binwrap": "^1.0.0"
+    "binwrap": "file:.."
```
Comment (michaelglass) [inline test_app/package.json]: when I ran "npm install ../" npm replaced the version here with this string

**Comment by avh4** [comment]
Sorry, I guess I'm still confused about what the issue is.. This branch creates a failure by locking a transitive dependency to a version that doesn't work, but isn't the fix just to not lock the dependency?  Also, aren't the package-lock.json files in binwrap ignored by the end-user projects when they install (and their own project-specific package-lock will be created when npm install runs?)  So I'm not convinced that the CI failure here is actually reproducing the original problem...

**Comment by michaelglass** [comment]
this issue should only occur for existing users of binwrap who already have a transitive dependency locked. When they upgrade to node 10, package installation starts to fail.

**Comment by avh4** [comment]
I tried the following with node 10:
  - install a package that uses binwrap
  - edit package-lock.json to change all references to natives-1.1.3 to natives-1.1.2
  - remove `node_modules`
  - run `npm install`

But doing that worked fine.  I thought that should fail on the last step if your hypothesis is correct.  Am I still misunderstanding something?

**Comment by michaelglass** [comment]
closing in favor of #15

**Pull request closed without merging**